src/main/java/mate/academy/internetshop/controller/RemoveItemFromBucketController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.internetshop.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.annotation.Inject;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ItemService;
+
+@WebServlet("/removeItem")
+public class RemoveItemFromBucketController extends HttpServlet {
+    @Inject
+    private static BucketService bucketService;
+    @Inject
+    private static ItemService itemService;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long itemId = Long.valueOf(req.getParameter("item_id"));
+        bucketService.removeItemFromBucket(0L, itemId);
```

Comment:
0L? 
is it magic number?
get bucket by userId

Reply:
I forgot to make a constant like I an other classes.